.gitignore

```diff
@@ -1 +1,2 @@
 node_modules
+.gitignore
```

Comment:
it must be not .gitignore but .DS_Store, isn't it ?

Reply:
tesi